### PR TITLE
fix: use specific dind version

### LINKED ISSUE #56: Use docker:20.10-dind instead of docker:dind for CI
For our current CI we should use the image `docker:20.10-dind`, not `docker:dind`.

`docker:dind` currently throws a permission error on build; this isn't the case with `docker:20.10-dind 

```
#8 11.48 make: /bin/sh: Operation not permitted
```

### PR DESCRIPTION
**Pull request by Ruesa18**
fixes #56

### CODE REVIEW

**Comment by Ruesa18** [comment]
Latest Pipeline: https://dev.whatwedo.ch/whatwedo/docker-base-images/-/pipelines/110103
winkgrin

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 image: docker:latest
 
 services:
-    - docker:dind
+    - docker:19.03-dind
```
Comment (s4mpl3d) [inline .gitlab-ci.yml:4]: @Ruesa18 best use `20.10-dind`. That's the same version we use in other pipelines

Comment (Ruesa18) [reply .gitlab-ci.yml:4]: @s4mpl3d we will take a look at this problem together tomorrow. Is that ok for you? 🙂

**Comment by Ruesa18** [comment]
This is already merged into the feature branch for version 2.5, so I'm gonna close this PR

**Pull request closed without merging**